packages/kotti-ui/source/kotti-filters/components/FilterRow.vue

```diff
@@ -1,5 +1,6 @@
 <template>
 	<KtForm
+		v-if="filter"
```

Comment:
This is going to break adding a new filter. When you click add Filter, we render the same component but the prop filter is null because we haven't yet chosen a column.

Reply:
then the v-if would have to be on the level of the fields

If the `filter` is null and the fields use formKeys, they will break as well --

@Juli0GT can u move the v-if to be around the content of the form

you can introduce a layer

like this 
```
<KtForm ....
  <template v-if="filter">
    <KtField* ...>
  </template>
```